docs/02.API-REFERENCE.md

```diff
@@ -757,6 +757,42 @@ jerry_eval (const jerry_char_t *source_p,
 - [jerry_create_external_function](#jerry_create_external_function)
 - [jerry_external_handler_t](#jerry_external_handler_t)
 
+## jerry_run_all_enqueued_jobs
+
+**Summary**
+
+Run enqueued Promise jobs until the first thrown error or until all get executed.
+
```

Comment:
will do (unfortunately, we're not consistent about this in the docs. `jerry_run` and `jerry_eval` have this note in the sources, too, but not in the docs. that's why I've skipped adding it for `jerry_run_all_enqueued_jobs`)